graphql/e2e/common/common.go

```diff
@@ -329,7 +329,14 @@ func containsRetryableCreateNamespaceError(resp *GraphQLResponse) bool {
 	return false
 }
 
-func CreateNamespace(t *testing.T, headers http.Header) uint64 {
+type CreateNamespaceParams struct {
+	GraphAdminURLs string
+	Count          int
+}
+
+func CreateNamespace(t *testing.T, headers http.Header, cnp ...CreateNamespaceParams) uint64 {
```

Comment:
we need to have two functions here `CreateNamespace` and `CreateNamespaces`.

Reply:
Resolved